README.md

```diff
@@ -0,0 +1,37 @@
+# My Porfolio
+
+A Porfolio to help showcase my skills and achievments to employers
+
```

Comment:
- [ ] It would be better if you add your project Screenshot in order to make your readme fancier 😉.